_events/0_A meeting template.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Awesome get-together on topic X 😎
+# Required. Displayed both on the event detail page and in the ICS calendar event.
+summary: 
+# Required. Displayed on the Events overview page.
+image: 
+# Optional. Displayed on Events overview page and the event detail page.
+permalink: 
+# Optional. "/events/:slug" recommended for recurring events, to avoid an an event getting a date-based URL.
```

Comment:
Does renaming then change the slug? Maybe the recommended value should be to explicitly write something?

Reply:
Renaming the file does not change the URL - it is based on the title: `/events/:year/:month/:title`. I think that's fine. The template already is quite complex, so I want to keep as few lines required or recommended as possible.

Reply:
I didn't mean adding additional recommendations. I was just thinking about changing the recommendation that is already there